File: NodeWorker/src/models/simple_node_execution_response.py
from src.models.node_execution_response import NodeExecutionResponse


class SimpleNodeExecutionResponse(NodeExecutionResponse):
    result_dataset_id: str

    def __init__(self) -> None:
        super().__init__()

    def set_result_dataset_id(self, id: str) -> None:
        self.result_dataset_id = id

    def get_result_dataset_id(self) -> str:
        return self.result_dataset_id

File: PipelineService/Models/MqttMessages/SimpleNodeExecutionResponse.cs
using System;

namespace PipelineService.Models.MqttMessages
{
    public class SimpleNodeExecutionResponse : NodeExecutionResponse
    {

        /// <summary>
        /// The id of the resulting dataset stored in the Dataset Storage.
        /// </summary>
        public Guid? ResultDatasetId { get; set; }
    }
}